src/index.js

```diff
@@ -1,12 +1,18 @@
 import express from 'express';
 import dotenv from 'dotenv';
+import swaggerUi from 'swagger-ui-express';
 import api from './api/routes';
 import globalMiddleware from './middleware/globalMiddleware';
+import  swaggerDoc from '../swagger.json';
+
 
 dotenv.config();
 const port = process.env.PORT || 3000;
 const app = express();
 
+app.use('/tesla-ah', swaggerUi.serve,swaggerUi.setup(swaggerDoc));
```

Comment:
Use the root URL `/` for the project documentation instead of `/tesla-ah`

Reply:
okay